controllers/user/api/v1/user_types.go

```diff
@@ -25,10 +26,10 @@ import (
 
 // UserSpec defines the desired state of User
 type UserSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
+	// display name of the user
 	DisplayName string `json:"displayName"`
+	// expiration time of the user
+	ExpirationDuration *string `json:"expirationDuration,omitempty"`
```

Comment:
Rename to `CSRExpirationSeconds` will match better, since here is config token expire, not user expire.